Content.Server/Silicons/StationAi/StationAiSystem.cs

```diff
@@ -73,4 +80,17 @@ private void AnnounceSnip(EntityUid entity)
         _chats.ChatMessageToMany(ChatChannel.Notifications, msg, msg, entity, false, true, filter.Recipients.Select(o => o.Channel));
         // Apparently there's no sound for this.
     }
+
+    private void OnSurveillanceCameraActivate(EntityUid camera, SurveillanceCameraComponent component, ref SurveillanceCameraActivateEvent args)
```

Comment:
Use Entity<>

Reply:
```suggestion
    private void OnSurveillanceCameraActivate(Entity<SurveillanceCameraComponent> camera, ref SurveillanceCameraActivateEvent args)
```
for example

you can get component by camera.Comp